autobuild/workflow.md

```diff
@@ -0,0 +1,216 @@
+1. fork the https://github.com/blocknetdx/blockchain-configuration-files repo
+2. clone the repo to your local machine
+3. checkout a branch from master with a suitable name, eg: bump-thing-v4.2.2 
+4. update information in the manifest-latest.json
+5. update information in autobuild/configs/thing.base.j2
+6. run app.py as described above to update the {wallet,xbridge}-confs/thing.conf
+7. commit the changes to
+   * manifest-latest.json
+   * autobuild/configs/thing.base.j2
+   * wallet-confs/thing.conf
+   * xbridge-confs/thing.conf
+8. push the branch back to your repo
+
+Other parts of the workflow depend on who is performing them and take place in other repos but are documented here for simplicity in the order they are expected to happen.
+
+## Build a new Docker image
+**Audience:** 
+Not directly applicable to 3rd party developers who are simply looking to list a coin because the workflow includes building a Docker image in the blocknetdx Docker hub repo. 3rd party developers could choose to branch the https://github.com/blocknetdx/dockerimages repo and adapt it to push the image to their own Docker hub repo, or reach out to Blocknet team members for assistance through [Discord #block-dx-listing](https://discord.gg/WnaygWwqFy) or a Github issue. 
+
+* Go to https://github.com/blocknetdx/dockerimages/actions and choose the BUILD IMAGE FROM TEMPLATE workflow. 
+* Click the *Run workflow* dropdown.
+* Choose the appropriate branch (most likely master) in the *Use workflow from* dropdown.
+* Enter the URL of your branch in the *Full path of a branch where the config can be found.* field, eg: https://raw.githubusercontent.com/walkjivefly/blockchain-configuration-files/bump-BLOCK-v4.3.3
+* Enter the full name of the coin you wish to create a Docker image for in the *The full name of a wallet. Example - bitcoin, dash, blocknet.* field
+* Enter the version you wish to build in *Tag or version of an image, please check manifest-latest.json. By default it takes the newest version.*
+* Click the green *Run workflow* button.
+
+If successful, the workflow takes around 30 minutes to build and push a new image to Docker hub. The image will have a -staging tag at this point signifying it has not been tested and released for public use.
+
+## Build exrproxy-env deployment package
+**Audience:**
+Not applicable to 3rd party developers who are simply looking to list a coin.
+
+* Follow the [Automated Docker-Based EXR Service Node Setup guide](https://docs.blocknet.co/service-nodes/setup/#automated-docker-based-exr-service-node-setup-recommended) to deploy one testnet Service Node and two testnet Trading Nodes on three different machines.
+* Configure each of these three nodes to support the coin to be tested and any counter-party coin you want to use to trade against the coin to be tested.
+* When you arrive at the step where ./app.py is called, be sure to pass it the (raw form) path to the branch you pushed to your Github Repo in step 8 above, like this:
```

Comment:
Sorry, I should've been more clear. I was requesting single back quotes around `./app.py` on this line (line 188), not within the code block.

Reply:
Doh, done.